sdk/azcore/internal/exported/request.go

```diff
@@ -169,3 +169,14 @@ func (req *Request) Clone(ctx context.Context) *Request {
 	r2.req = req.req.Clone(ctx)
 	return &r2
 }
+
+// not exported but dependent on Request
```

Comment:
Then could it somewhere else instead, like `arm/internal`?

Reply:
Right now it's only used in `runtime`, but it will also be used in `arm/runtime` as part of the tracing work.

Reply:
Would it create an import cycle in `shared`? (I can live with it being here if there's no better alternative)

Reply:
Unfortunately yes as that's where I originally had placed it.